extern/acts/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ message( STATUS "Building Acts as part of the TRACCC project" )
 
 # Declare where to get Acts from.
 set( TRACCC_ACTS_SOURCE
-   "URL;https://github.com/krasznaa/acts/archive/refs/tags/v10.0.1.tar.gz;URL_MD5;fcd2f2f7b5810e34b8faa12634043be3"
+   "GIT_REPOSITORY;https://github.com/acts-project/acts.git;GIT_TAG;v21.1.0"
```

Comment:
You should use the TGZ file of the `v21.1.0` tag here.

https://github.com/acts-project/acts/archive/refs/tags/v21.1.0.tar.gz

Setting the appropriate MD5 hash for it along the way.